.travis.yml

```diff
@@ -1,24 +1,18 @@
 language: cpp
 
 before_install:
-  - if [ "$INSTALL" = "true" ]; then
-      sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test;
-      sudo apt-get update -qq;
-      sudo apt-get install -qq -y --force-yes gcc-4.8 g++-4.8;
-    fi
```

Comment:
This has been removed because made redundant by specifying the packages, correct?